conf/messages.en

```diff
@@ -38,8 +38,12 @@ error.number = Please enter a valid number
 error.required = Please enter a value
 error.summary.title = There is a problem
 
-categoryGuidance.title = Setting up your profile
-categoryGuidance.h1 = Boo
+categoryGuidance.title = Categorisation - Trader Goods Profile
```

Comment:
I think this will cause the same issue as Nidhi found in the other ticket, where "- Trader Goods Profile" becomes duplicated in the title. I think we only need Categorisation here

Reply:
Changed